src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -526,30 +529,41 @@ public static IEnumerable<object[]> PostServersStreamsAndExpectedData
         [Theory, MemberData("PostServers")]
         public async Task PostAsync_CallMethod_NullContent(Uri remoteServer)
         {
-            using (var handler = new HttpClientHandler())
+            using (var client = new HttpClient())
             {
-                using (var client = new HttpClient(handler))
-                {
-                    HttpContent obj = new StringContent(String.Empty);
-                    HttpResponseMessage response = await client.PostAsync(remoteServer, null);
-                    Assert.Equal(HttpStatusCode.OK, response.StatusCode);
-                    var responseContent = await response.Content.ReadAsStringAsync();
-                    Assert.True(JsonMessageContainsKeyValue(responseContent, dataKey, String.Empty));
-                    _output.WriteLine(responseContent);
-                }
+                HttpContent obj = new StringContent(String.Empty);
+                HttpResponseMessage response = await client.PostAsync(remoteServer, null);
+                Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+                var responseContent = await response.Content.ReadAsStringAsync();
+                Assert.True(JsonMessageContainsKeyValue(responseContent, dataKey, String.Empty));
+                _output.WriteLine(responseContent);
             }
         }
 
         [Fact]
         public async Task PostAsync_IncorrectUri_MethodNotAllowed()
         {
-            using (var handler = new HttpClientHandler())
+            using (var client = new HttpClient())
             {
-                using (var client = new HttpClient(handler))
-                {
-                    HttpResponseMessage response = await client.PostAsync(HttpTestServers.RemoteGetServer, null);
-                    Assert.Equal(HttpStatusCode.MethodNotAllowed, response.StatusCode);
-                }
+                HttpResponseMessage response = await client.PostAsync(HttpTestServers.RemoteGetServer, null);
+                Assert.Equal(HttpStatusCode.MethodNotAllowed, response.StatusCode);
+            }
+        }
+
+        [Fact]
+        [ActiveIssue("Channel binding not implemented on *Nix", PlatformID.AnyUnix)]
```

Comment:
I believe we need to put in a GitHub issue instead of a string? (Maybe CI or other systems might break if they see this.)